### PR TITLE
Character limit issue on cases

### PR DESCRIPTION
**Pull request by erikh360**
This moves the character limits into the settings_common.py.

Fields size limits are from db and will be handled by frontend.

### CODE REVIEW

**Comment by erikh360** [comment]
@miltontony please review. It's a big one, the max length was used more than I initially thought. 

Also, my editor removed some empty spaces in the test coffee scripts that I updated, it's better in the long run but makes this review more complicated, let me know if I should revert them.

File: casepro/cases/views.py [hunk 1]
```diff
@@ -24,7 +24,7 @@
 from casepro.utils import month_range, humanize_seconds
 from casepro.utils.export import BaseDownloadView
 
-from . import MAX_MESSAGE_CHARS
+from casepro.settings_common import MAX_MESSAGE_CHARS
```
Comment (miltontony) [inline casepro/cases/views.py]: The Django way is to import this from settings
```python
from django.conf import settings
...
context['max_msg_chars'] = settings.MAX_MESSAGE_CHARS
```

File: casepro/profiles/context_processors.py [hunk 2]
```diff
@@ -1,4 +1,5 @@
 from __future__ import absolute_import, unicode_literals
+from casepro.settings_common import MAX_MESSAGE_CHARS, CASE_SUMMARY_MAX_LEN
```
Comment (miltontony) [inline casepro/profiles/context_processors.py]: Same here - use `django.conf.settings`

File: static/coffee/controllers.coffee [hunk 3]
```diff
@@ -12,9 +12,7 @@ INTERVAL_CASE_TIMELINE = 30000
 INFINITE_SCROLL_MAX_ITEMS = 1000
 
 # Form constraints
-CASE_SUMMARY_MAX_LEN = 255
 CASE_NOTE_MAX_LEN = 1024
-OUTGOING_TEXT_MAX_LEN = 480
```
Comment (miltontony) [inline static/coffee/controllers.coffee]: I see you've removed this, I can't see what you replaced it with.

File: casepro/settings_common.py [hunk 4]
```diff
@@ -56,6 +56,8 @@
 SITE_HIDE_CONTACT_FIELDS = []  # Listed fields should not be displayed
 SITE_CONTACT_DISPLAY = "name"  # Overrules SITE_HIDE_CONTACT_FIELDS Options: 'name', 'uuid' or 'urns'
 SITE_ALLOW_CASE_WITHOUT_MESSAGE = True
+MAX_MESSAGE_CHARS = 640
+CASE_SUMMARY_MAX_LEN = 8000
```
Comment (miltontony) [inline casepro/settings_common.py]: I think their default was `255` - let's use that.

**Comment by erikh360** [comment]
@miltontony ready for review again, thanks.

File: static/coffee/controllers.coffee (hunk 3 above)
Comment (miltontony) [inline static/coffee/controllers.coffee]: 👌

**Comment by miltontony** [comment]
👍

**Pull request merged**